storage/entities/account.js

```diff
@@ -266,20 +283,20 @@ class Account extends BaseEntity {
 
 		this.addFilter('votes', ft.CUSTOM, {
 			condition:
-				'mem_accounts.address IN (SELECT "accountId" FROM mem_accounts2delegates WHERE "dependentId" = ${votedFor})',
+				'mem_accounts.address IN (SELECT "accountId" FROM mem_accounts2delegates WHERE "dependentId" = ${votes})',
 		});
 		this.addFilter('votes_in', ft.CUSTOM, {
 			condition:
-				'mem_accounts.address IN (SELECT "accountId" FROM mem_accounts2delegates WHERE "dependentId" IN (${votedFor_in:csv}))',
+				'mem_accounts.address IN (SELECT "accountId" FROM mem_accounts2delegates WHERE "dependentId" IN (${votes_in:csv}))',
 		});
 
 		this.addFilter('members', ft.CUSTOM, {
```

Comment:
As discussed before, let's take the opportunity to use more meaningful names like `membersPublicKeys`